src/frontend/src/components/dappList.ts

```diff
@@ -0,0 +1,119 @@
+import { html, TemplateResult } from "lit-html";
+
+export type DappDescription = {
+  id: string;
+  name: string;
+  oneLiner?: string;
+  link: string;
+  derivationOrigin?: string;
+  description: string;
+  logo: string;
```

Comment:
can we simply reuse the dapp name as path for the logo?

Reply:
I think its a good idea to keep it separated since the name could contain special characters.